src/nvim/shada.c

```diff
@@ -2200,8 +2207,22 @@ static inline ShaDaWriteResult shada_read_when_writing(ShaDaReadDef *const sd_re
               }
               shada_free_shada_entry(&wms_entry->data);
             }
+          } else {
+            FOR_ALL_BUFFERS(buf) {
+              if (buf->b_ffname != NULL && path_fnamecmp(entry.data.filemark.fname, buf->b_ffname) == 0) {
```

Comment:
Lint failure because this line is too long
```suggestion
              if (buf->b_ffname != NULL
                  && path_fnamecmp(entry.data.filemark.fname, buf->b_ffname) == 0) {
```

Reply:
Oops my bad. Thanks!